src/misc/cross.cpp

```diff
@@ -782,6 +782,46 @@ std::string cfstr_to_string(CFStringRef source)
 	return lang;
 }
 
+void fat_attributes_to_xattr(uint16_t fattr, char* outbuf, size_t outsize)
+{
+	memset(outbuf, 0, outsize);
+
+	strcat(outbuf, "\"");
```

Comment:
Quotes aren't needed.  Sorry about the confusion on Discord.  The CLI tool will show it as a string if the value contains only ASCII data.